src/main/resources/db/changelog/changes/08-insert-shopping-cart-for-admin-user.yaml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: your_name
+      changes:
+        - insert:
+            tableName: shopping_carts
+            columns:
+              - column:
```

Comment:
why do you need this changelog?

Reply:
As we added admin user manually we will get exception if admin decide to use shop cart